### ISSUE TITLE
Error running on VOT toolkit, trax did not response.

### ISSUE THREAD

**Issue by soldatjiang** [ihpdep/LDES#5]
Have you tested your run_vot.m script on vot toolkit? It ran into error and I can't debug it.

**Comment by ihpdep** [comment]
Yes, I tested it on Vot-2015. It should be fine. Please make sure you have the latest code cause I fixed some bugs last month.
